CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.8)
```

Comment:
needs to be updated to 3.12 to understand `CMAKE_CXX_STANDARD=20`

Reply:
done